crates/cxx-qt-lib/src/types/qpoint.cpp

```diff
@@ -7,6 +7,21 @@
 // SPDX-License-Identifier: MIT OR Apache-2.0
 #include "cxx-qt-lib/include/qpoint.h"
 
+#include "assertion_utils.h"
+
+#include <cstdint>
+
+// QPoint has "int" members - xp and yp
+// Rust represents these as 32-bit integer types.
+// https://codebrowser.dev/qt5/qtbase/src/corelib/tools/qpoint.h.html#QPoint::xp
```

Comment:
i was using `code.qt.io` rather than `codebrowser.dev` as the canonical source (as codebrowser.dev just moved and we don't know if it'll move/change again), and pinned to either `v5.15.6-lts-lgpl` or `v6.2.4` so that the code doesn't move/change.

Reply:
The new links do have the advantage of not pinning to a specific line though.
So these links should be stable :thinking:

Reply:
But the tag will never change, v6.2.4 will always have a member called xp. Whereas your link will fail if they change QPoint to be `xpos` as it's not pinned to a specific Qt release

Reply:
Updated them to perma-links